pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -397,8 +398,7 @@ func newPrefixWriter(out io.Writer, prefix string) io.Writer {
 	return writer
 }
 
-func (o *MustGatherOptions) waitForPodRunning(pod *corev1.Pod) error {
-	phase := pod.Status.Phase
+func (o *MustGatherOptions) waitForMustGatherToComplete(pod *corev1.Pod) error {
```

Comment:
nit: `waitForGatherToComplete` seems much appropriate if container is called `gather`